tinybytes/src/bytes_string.rs

```diff
@@ -106,6 +106,13 @@ impl BytesString {
         // SAFETY: We assume all BytesStrings are valid UTF-8.
         unsafe { std::str::from_utf8_unchecked(&self.bytes) }
     }
+
```

Comment:
As trivial as it may seem, we should really have a unit test for any public methods.